Formula/alerter.rb

```diff
@@ -0,0 +1,25 @@
+class Alerter < Formula
+  desc "Send User Alert Notification on macOS from the command-line"
+  homepage "https://github.com/vjeantet/alerter"
+  url "https://github.com/vjeantet/alerter/archive/refs/tags/004.tar.gz"
+  version "4"
+  sha256 "c4c16735e1a57ce04a5acfa762bd44ab8ef10884308725ab886b984b1de17bfe"
+  license "MIT"
+
+  depends_on xcode: :build
+
+  def install
+    xcodebuild "-arch", Hardware::CPU.arch,
+                "-project", "alerter.xcodeproj",
+                "-target", "alerter",
+                "-configuration", "Release",
+                "SYMROOT=build",
+                "CODE_SIGN_IDENTITY=",
+                "MACOSX_DEPLOYMENT_TARGET=10.9"
```

Comment:
```suggestion
                "MACOSX_DEPLOYMENT_TARGET=#{MacOS.version}"
```

Reply:
Thank you... Is there a list of all the variables we can use somewhere